DigitalOffice/Services/FileService/Models/FileRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace FileService.Models
+{
+    public class FileRequest
+    {
+        public Guid Id { get; set; }
```

Comment:
As far as I know, the request should not contain a unique file identifier. We issue it ourselves when writing to the database.

Reply:
As I mentioned above, we do not have Response model (at least I havent bump into any of them yet) so this FileRequest (I should have named it just File *sigh*) is a request and a response.
When we get the list of files, we get their data and their Ids so that we could click them and, for example, download by Id.

Reply:
Fixed Request model, removed Id property and created a separate Dto/Response model called "File" in "Models/" folder.